test/WebJobs.Script.Tests/SamplesEndToEndTests.cs

```diff
@@ -617,6 +621,14 @@ private bool IsHostRunning()
 
             public void Dispose()
             {
+                var manager = _config.DependencyResolver.GetService<WebScriptHostManager>();
```

Comment:
Funny - I was debugging this tonight and realized that due to recent changes, the WebScriptHostManager is no longer disposed automatically when the HttpServer is Disposed. That's due to recent changes here: https://github.com/Azure/azure-webjobs-sdk-script/blob/dev/src/WebJobs.Script.WebHost/App_Start/AutofacBootstrap.cs#L14, where the instance was marked externally owned.

If these changes get your tests passing great, but I'll also be revisiting that code to make sure things are disposed properly.